core/src/com/mygdx/game/client/model/GameState.java

```diff
@@ -0,0 +1,22 @@
+package com.mygdx.game.client.model;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import java.util.ArrayList;
+import java.util.List;
+
+@Singleton
+@Setter
+@Getter
+public class GameState {
+
+    private List<Field> fieldList;
```

Comment:
Nie wiem jaki jest cel tej struktury danych na razie. W sensie spodziewałem się, że będziemy mogli się do fieldów odwoływać jakoś po x i y, a tutaj jest lista

Reply:
Zrobić jakąś mapę? I klasę Coordinates która by była kluczem?

Reply:
Fajny pomysł z tym kluczem jako Coordinates. Byłoby bardziej opisowo niż tylko Vector2.

I w sumie to tak, tak byłoby ok.